packages/cyberstorm/src/components/Layout/PackageListings/PackageListings.tsx

```diff
@@ -79,6 +79,8 @@ export default function PackageListings(props: PackageListingsProps) {
     if (!isEqual(updatedAvailableCategories, filters.availableCategories)) {
       filters.setAvailableCategories(updatedAvailableCategories);
     }
+
+    filters.setPageObjectCount(datas.length);
```

Comment:
Why do this here? This effect is for updating categories, and page count is totally unrelated. I'd do this in a separate effect, or the one that fetches the data above.

Also when adding stuff to effects, update the dependency list argument as well.

Reply:
Checked this component a bit more while working on Dapper refactoring. If I'm not mistaken this approach won't work anyway since `datas.length` contains only the returned packages, i.e. is limited by the requested "page size" (20 packages per page). Backend needs to return the total number of packages separately, which it currently does not do (at least according to Dapper interface/implementations).